packages/utils/package.json

```diff
@@ -42,15 +42,14 @@
   },
   "devDependencies": {
     "@edx/browserslist-config": "1.1.0",
-    "@edx/frontend-build": "12.3.0",
-    "@edx/frontend-platform": "1.15.6",
+    "@edx/frontend-build": "12.7.0",
+    "@edx/frontend-platform": "4.0.1",
     "react": "16.14.0",
     "react-dom": "16.14.0",
     "react-router-dom": "5.2.0",
     "react-test-renderer": "16.14.0"
   },
   "peerDependencies": {
-    "@edx/frontend-platform": "^1.9.6 || ^2.0.0",
```

Comment:
[clarification] Why are we removing this peer dependency? `@edx/frontend-platform` is used by the `utils` package and is up to the MFE (consumer) to install, not this package. This is why `@edx/frontend-platform` is a `devDependency`, not a `dependency`.

It should denote the full range of versions of `@edx/frontend-platform` that `utils` is compatible with. If one of these existing versions is removed from the existing peer dependency list, this package needs to be released as a breaking change.